backend/plugin/db/oracle/sync.go

```diff
@@ -39,9 +44,146 @@ func (driver *Driver) SyncInstance(ctx context.Context) (*db.InstanceMetadata, e
 }
 
 // SyncDBSchema syncs a single database schema.
-func (*Driver) SyncDBSchema(_ context.Context, database string) (*storepb.DatabaseMetadata, error) {
+func (driver *Driver) SyncDBSchema(ctx context.Context, databaseName string) (*storepb.DatabaseMetadata, error) {
+	// TODO(d): filter out system tables.
+	txn, err := driver.db.BeginTx(ctx, nil)
```

Comment:
Should we use read-only txn?

Reply:
Seems like readonly isn't supported option :(